.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,4 @@
-### Commit Message
-COMMIT/PR TITLE HERE (must follow conventionalcommits.org)
-
-COMMIT MESSAGE HERE
-### End Commit Message
+Title & body will be used as commit message and MUST follow conventional commits (see contribution guide).
```

Comment:
Do we really need this now? If the title is used as the first line, we already have validation on that. 

Pretty sure people will forget to delete this line.

Reply:
Fair enough. Let’s just remove